caikit_nlp/modules/text_generation/text_generation.py

```diff
@@ -153,22 +153,21 @@ def save(self, model_path):
             model_path: str
                 Folder to save text-generation caikit model
         """
-        saver = ModuleSaver(
-            self,
-            model_path=model_path,
-        )
+        saver = ModuleSaver(self, model_path=model_path)
         with saver:
-            artifacts_dir = "artifacts"
             saver.update_config(
                 {
-                    "artifact_path": artifacts_dir,
+                    "base_model_name": self.base_model_name,
                     "bos_token": self._bos_token,
                     "sep_token": self._sep_token,
                     "eos_token": self._eos_token,
                     "pad_token": self._pad_token,
                 }
             )
             if self.base_model:
+                artifacts_dir = "artifacts"
+                log.debug("Saving model artifacts to %s", artifacts_dir)
+                saver.update_config({"artifact_path": artifacts_dir})
```

Comment:
I know this is kind of residual from what was previously there, but I wonder if this should be renamed, since it's pretty nondescriptive

Reply:
I think this would be a breaking change with existing models. There aren't any real ones that I know of at this point, but I also don't think `artifact_path` is too bad (it's the path to the model artifacts!), so I'm inclined to leave it.